/Users/alist/Documents/MIT/Classes/Spring19/6835/MMCheckComp/venv/bin/python "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydevd.py" --multiproc --qt-support=auto --client 127.0.0.1 --port 51933 --file /Users/alist/Documents/MIT/Classes/Spring19/6835/MMCheckComp/Step1_preprocess.py
pydev debugger: process 40668 is connecting

Connected to pydev debugger (build 182.4505.26)
a -1.600098:4.738281, q -0.637207:3.775391, e -128.000000:255.000000
a -3.364746:5.992187, q -2.579102:5.206543, e -128.000000:255.000000
a -2.948730:7.397949, q -1.242676:5.691895, e -128.000000:255.000000
a -1.031738:2.244141, q -1.031738:2.244141, e -128.000000:255.000000
a -1.369629:2.684570, q -1.035156:2.350098, e -128.000000:255.000000
a -1.138184:2.428223, q -0.729980:2.020020, e -128.000000:255.000000
a -4.426270:10.101562, q -2.305664:7.980957, e -128.000000:255.000000

Process finished with exit code 0
